### ISSUE TITLE
[BUG]: Trivial Abseil logging code fails to link

### ISSUE THREAD

**Issue by benschreiber** [compiler-explorer/compiler-explorer#6722]
### Describe the bug

I selected the Abseil library, included `absl/log/log`, and wrote a Hello World type program that uses Abseil's `LOG` macro.
The object file was generated, which indicates that the header is being found correctly.
Linking fails with
```
undefined reference to `absl::log_internal::LogMessage::LogMessage(char const*, int, absl::log_internal::LogMessage::InfoTag)'
```

### Steps to reproduce

Enable "Abseil - Trunk" from the list of libraries.
```
#include "absl/log/log.h"

int main() {
    LOG(INFO) << "Hello, Abseil logging!";
    return 0;
}
```
Tested and failure observed on recent clang and gcc.

### Expected behavior

Compilation should succeed.
Execution should print a time/date/location prefix and "Hello, Abseil logging!".

### Reproduction link

https://godbolt.org/z/nrhe73Tzb

### Screenshots

Not applicable

**Comment by partouf** [comment]
Unfortunately this is not the first time. The people at Abseil seem to keep changing the source layout every 5 seconds and we need to keep the binaries and linking manually up to date. It's pretty much not feasible for us to keep everything up to date. It would be nice to have an abseil contributor help out, or something.
https://github.com/compiler-explorer/compiler-explorer/issues/6633
https://github.com/compiler-explorer/compiler-explorer/issues/5982

**Comment by benschreiber** [comment]
Abseil appears to do LTS branch releases. Could godbolt add those to the library version picker alongside "Trunk", which may be more likely to have breaking changes?

**Comment by jpan127** [comment]
@partouf Agreed with @benschreiber , is there any way to do that similar to how other libraries (`fmt` for example)?

I only want to test with `absl` once in awhile, but seems like every time I come back to it it's a new failure because of `trunk` 😢 .

**Comment by sibi-venti** [comment]
https://github.com/compiler-explorer/compiler-explorer/issues/7243

**Comment by partouf** [comment]
We have removed Abseil trunk and replaced it with version 20250127.0 and fixes this issue
